oarepo_ui/theme/assets/semantic-ui/js/oarepo_ui/forms/util.js

```diff
@@ -21,6 +24,7 @@ export function parseFormAppConfig(rootElementId = "form-app") {
   const recordPermissions = getInputFromDOM("record-permissions");
   const files = getInputFromDOM("files");
   const links = getInputFromDOM("links");
+  formConfig.getFieldData = getFieldData(formConfig.ui_model);
```

Comment:
Why doing it like this, if FieldDataProvider does basically the same thing? https://github.com/oarepo/oarepo-ui/pull/206/files?diff=unified&w=1#diff-2beecbc1422ee475e5299c154a15418f2e8f6bad914a1800cd9881096bebc64bR28

Reply:
@mirekys This is artifact from initial implementation, you are right. My bad